### ISSUE TITLE
Labels overlapping when zooming in bar chart

### ISSUE THREAD

**Issue by JulianWgs** [plotly/plotly.py#2669]
Hello,

when zooming horizontally on the right bar column the labels of the left plot overlap. Here is a minimum working example and a screenshot.

I'm using plotly 4.9.0 on anaconda for Windows in Jupyterlab.

```python
df = pd.DataFrame({
    "x": ["very_long_name_a", "very_long_name_a", "very_long_name_b", "very_long_name_c"],
    "y": [1, 2, 1, 4],
    "facet_col": ["z", "y", "z", "y"]
})
px.bar(df, x="x", y="y", facet_col="facet_col")
```

![grafik](https://user-images.githubusercontent.com/31596773/88516763-103fbc80-cfee-11ea-8c55-337aec4a0a4a.png)

**Comment by nicolaskruchten** [comment]
Thanks for the bug report! We'll fix this in the JS repo: https://github.com/plotly/plotly.js/issues/5063

**Comment by gvwilson** [comment]
Hi - we are trying to tidy up the stale issues and PRs in Plotly's public repositories so that we can focus on things that are still important to our community. Since this one has been sitting for several years, I'm going to close it; if it is still a concern, please add a comment letting us know what recent version of our software you've checked it with so that I can reopen it and add it to our backlog. Thanks for your help - @gvwilson

**Comment by JulianWgs** [comment]
This is indeed fixed in 5.22.0. Thank you!
